charts/kube-otel-stack/values.yaml

```diff
@@ -126,7 +126,14 @@ tracesCollector:
       pipelines:
         traces:
           receivers: [otlp]
-          processors: [memory_limiter, resource, resourcedetection/env, k8sattributes, batch]
+          processors:
+            [
+              memory_limiter,
+              resource,
+              resourcedetection/env,
+              k8sattributes,
+              batch,
+            ]
```

Comment:
this isn't the correct formatting for a helm list

Reply:
Technically YAML sequences can be split like this using newlines, but I will turn it into the usual list syntax for consistency sake.